rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -251,6 +254,43 @@ def _create_networks(self, network_create_args):
         return lswitches
 
 
+    @atomic.action_timer("ovn_network.of_check_port")
+    def _of_check_ports(self, lports, sandboxes, port_bind_args):
```

Comment:
@hzhou8 Thanks for your response.

1. I am very happy to give the first shot and will try to submit a PR to #97 soon.
2. Putting flow verification as an option in the post-test stage sounds good to me.